packages/pixeloven/cli-addon-storybook/src/commands/story.ts

```diff
@@ -1,32 +1,36 @@
-import { AddonStorybookRunContext, StorybookExecutionType } from "../types";
+import { AddonStorybookToolbox, StorybookExecutionOptionTypes, StorybookExecutionType } from "../types";
 
 export default {
     alias: ["--story", "-s"],
     name: "story",
-    run: async (context: AddonStorybookRunContext) => {
-        const { parameters, pixelOven, storybook } = context;
-        if (!parameters.first) {
-            pixelOven.invalidArgument("Must provide a story run type.");
+    run: async (toolbox: AddonStorybookToolbox) => {
+        const { parameters, pixelOven, storybook } = toolbox;
+        const task = parameters.first;
+        if (!task) {
+            pixelOven.invalidArgument("Must provide a task for Storybook to run.");
+            pixelOven.exit("Storybook", 1);
```

Comment:
@ductiletoaster I decided it's cleaner to keep the exit and invalidArg separate because otherwise we'd have to pass the task and exit code to the invalid argument method or the message and argument to the exit method.

Reply:
Makes sense.